sample_tracking/app/sign-up-requests/page.tsx

```diff
@@ -111,8 +111,10 @@ export default function SignUpRequests() {
         const dateString = `${date.getMonth() + 1} ${date.getDate()} ${date.getFullYear()}`;
         const orgId = getRanHex(20);
         const newOrgRef = doc(db, "organizations", orgId);
+        const orgEmail = `${orgName}@timberid.org`;
```

Comment:
i am a little confused as to how a org@timberid.org email can be useful to an organization.  Are we setting up a gmail account for the org that they can log into somewhere?

Reply:
This was requested by Kader for EE. I'm not exactly sure how he uses it but its required for things on his end.

Reply:
ahh ok -- he uses it for folder permissions in EE.  makes sense